parsons/phone2action/p2a.py

```diff
@@ -29,33 +30,26 @@ def __init__(self, app_id=None, app_key=None):
         self.app_id = check_env.check('PHONE2ACTION_APP_ID', app_id)
         self.app_key = check_env.check('PHONE2ACTION_APP_KEY', app_key)
         self.auth = HTTPBasicAuth(self.app_id, self.app_key)
-        self.uri = PHONE2ACTION_URI
-
-    def _request(self, url, args=None):
-        # Internal request method
-
-        r = requests.get(url, auth=self.auth, params=args)
-        r.raise_for_status()
```

Comment:
Any issues here with removing the raise for status and the validate response?

Reply:
Since we are now leveraging the `APIConnector`, we get that functionality from the `APIConnector`'s `get_request` / `post_request` methods.

Reply:
My mistake. All good!